src/app/pages/login/login.page.ts

```diff
@@ -71,6 +72,9 @@ export class LoginPage extends BasePageComponent implements OnInit {
     try {
       await this.platform.ready();
       await this.initialiseAppConfig();
+
+      // this.store$.dispatch(StartSendingLogs());
```

Comment:
`StartSendingLogs` and `LoadEmployeeName` will be reenabled once the AppConfig is implemented properly.